Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository = "https://github.com/VersBinarii/xpt2046"
 name = "xpt2046"
 
 [dependencies]
-embedded-hal = "=1.0.0-alpha.7"
+embedded-hal = "^1.0.0"
```

Comment:
This should actually b = "1.0.0" per https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements